openfecli/fetchables.py

```diff
@@ -5,6 +5,20 @@
 _EXAMPLE_NB_BASE = ("https://raw.githubusercontent.com/"
```

Comment:
If we change the url here to be against a tag/commit then we can't accidentally break these. If we are patching the files, it makes more sense to be intentional about updating this url to reflect that.

Reply:
@richardjgowers: Is the intent for `try.openfree.energy` to only update on releases? Like, a typo fix in  notebook will only be reflected on a release of `openfe`? 

If we do that, I'd recommend setting up some automation to check that that this is the most up-to-date. "Be intentional about updating the URL" sounds to me like "we're going to forget to update the URL."

Reply:
With the installer hard-coded + the URL using a tagged release, then we are pretty set in keeping `try.openfree.energy` stable. Even if we push a new release and have API changes, the notebook will still be using the "old" installer, so it will stay stable and just have to remember to update it.

Reply:
Yes I think `try.` should be off releases not HEAD